gradle.properties

```diff
@@ -8,17 +8,20 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
-pravegaVersion=0.4.0
-flinkConnectorVersion=0.4.0
+pravegaVersion=0.5.0-2223.ce3ba81-SNAPSHOT
+flinkConnectorVersion=0.5.0-75.01ba05e-SNAPSHOT
 
 ### Pravega-samples output library
 samplesVersion=0.5.0-SNAPSHOT
 
 ### Flink-connector examples
```

Comment:
The title could be something like `Flink dependencies`, right?

Reply:
fixed